src/codeu/chat/client/commandline/Chat.java

```diff
@@ -149,14 +199,26 @@ public void invoke(List<String> args) {
     panel.register("u-add", new Panel.Command() {
       @Override
       public void invoke(List<String> args) {
-        final String name = args.size() > 0 ? args.get(0) : ""; //this works if the args list passed in contains tokens
+        final String name = args.size() > 0 ? String.join(" ", args.subList(0, args.size())) : "";
```

Comment:
Isn't `args.subList(0, args.size())` equivalent to `args`? ie, this seems the same as just doing `final String name = String.join(" ", args);`.